05/DB.test.js

```diff
@@ -1 +1,84 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+
+describe('DB -> insert', () => {
+    it('return data when ID equals number', () => {
+        expect.assertions(1);
+
+        const db = new DB();
+
+        const data = {id: 1}
+        const promise = db.insert(data);
+
+        return promise.then(res => {
+            expect(res).toBe(data);
+        });
+    });
+
+    it('reject if ID is duplicated', async () => {
+        expect.assertions(1);
+
+        const db = new DB();
+
+        const data = {id: 1};
+        const newData = {id: 1};
+
+        await db.insert(data);
+
+        const promise = db.insert(newData);
+
+        return promise.catch(err => {
```

Comment:
Jeśli używamy async/await to warto się już trzymać tej formy do końca.
https://jestjs.io/docs/asynchronous#asyncawait